lib/redcord/attribute.rb

```diff
@@ -54,6 +55,11 @@ def index_attribute(attr, type)
         class_variable_get(:@@index_attributes) << attr
       end
     end
+    
+    sig { params(index_name: Symbol, attrs: T::Array[Symbol]).void }
+    def custom_index(index_name, attrs)
+      class_variable_get(:@@custom_index_attributes)[index_name] = attrs
```

Comment:
is there a length limit on the items in a zset?

Reply:
Set is a collection of normal strings, and _A String value can be at max 512 Megabytes in length._